src/WebSdk/Publish/Targets/PublishProfiles/DefaultMSDeploy.pubxml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <WebPublishMethod>MSDeploy</WebPublishMethod>
+    <LastUsedBuildConfiguration>Release</LastUsedBuildConfiguration>
+    <LastUsedPlatform>Any CPU</LastUsedPlatform>
+    <SkipExtraFilesOnServer>True</SkipExtraFilesOnServer>
+    <MSDeployPublishMethod>WMSVC</MSDeployPublishMethod>
+    <EnableMSDeployBackup>True</EnableMSDeployBackup>
+  </PropertyGroup>
```

Comment:
Do you need to include '<PublishProvider>AzureWebsite</PublishProvider>' here, too?

Reply:
I haven't added for MSDeploy because it could be an onprem server with MSDeploy.